_topic_maps/_topic_map.yml

```diff
@@ -3484,6 +3484,8 @@ Topics:
     File: serverless-functions-setup
   - Name: Getting started with functions
     File: serverless-functions-getting-started
+  - Name: Building and deploying functions on-cluster
```

Comment:
Is "on-cluster" a term that is commonly used in the serverless docs? Just sounded odd when I read it - wasn't sure if "on the cluster" would be the same/better. Also just wasn't sure what the alternative option is (as opposed to on the cluster) as I don't see another assembly that points out a different way (and I don't see in the first paragraph what the other way is).